constraints.txt

```diff
@@ -109,6 +110,7 @@ guppy3==3.1.2
 h5netcdf==0.12.0
 h5py==3.6.0
 hdfs==2.6.0
+heapdict==1.0.1
```

Comment:
BSD 3-clause.

Not quite sure what triggered this dep... I don't see it as a requirement for dask-distributed.